### PR TITLE
[#3] [iOS] [UI] As a user, I can see a Splash screen when app starts

### LINKED ISSUE #3: [iOS] [UI] As a user, I can see a Splash screen when app starts
# Why

To build the application identity and keeps users occupied while your app loads in the background, we will show the `Splash` screen when the app is starting.

# Acceptance Criteria
- Show a background image and the `Nimble` as the following design

Image

[Figma](https://www.figma.com/file/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=31-0&t=lF1bJOT4aAdBAohd-0)

### PR DESCRIPTION
**Pull request by phongvhd93**
- Close #3 

## What happened 👀

- Create SplashView for the splash screen.
- Add a background image and nimble logo as the image resource

## Insight 📝

- When the user opens the app, a splash screen will be shown to keep them occupied while background jobs like fetching other data from API are running.

## Proof Of Work 📹

https://user-images.githubusercontent.com/22606906/231658019-9b0c5f43-adb5-4759-a0ea-ee57d36dc929.mp4

### CODE REVIEW

File: iosApp/PhongKMMIC/Sources/Presentation/Modules/Splash/SplashView.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+//
+
+import SwiftUI
+
+struct SplashView: View {
+
+    @State private var isLoaded = false
+
+    var body: some View {
+        GeometryReader { geometry in
```
Comment (markgravity) [inline iosApp/PhongKMMIC/Sources/Presentation/Modules/Splash/SplashView.swift]: We can omit this `GeometryReader` here 🙏

**Review by markgravity — COMMENTED** [review]
The rest LGTM

File: iosApp/PhongKMMIC/Sources/Presentation/Modules/Splash/SplashView.swift (hunk 1 above)
Comment (phongvhd93) [reply iosApp/PhongKMMIC/Sources/Presentation/Modules/Splash/SplashView.swift]: @markgravity, would you fill me in? Why can we omit `GeometryReader`? 🤔

Comment (markgravity) [reply iosApp/PhongKMMIC/Sources/Presentation/Modules/Splash/SplashView.swift]: @phongvhd93 You can check my approach [here](https://github.com/markgravity/kmm-ic/pull/56), mostly the same as you 😁

File: iosApp/PhongKMMIC/Sources/Application/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import UserNotifications
+
+final class AppDelegate: NSObject, UIApplicationDelegate, UNUserNotificationCenterDelegate {
+
+    func userNotificationCenter(
+        _ center: UNUserNotificationCenter,
+        willPresent notification: UNNotification,
+        withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void
+    ) {
+        completionHandler([.banner, .sound])
```
Comment (suho) [inline iosApp/PhongKMMIC/Sources/Application/AppDelegate.swift]: Why do we need user notifications?

**Comment by markgravity** [comment]
@phongvhd93 Please rebase the PR 🙏

File: iosApp/PhongKMMIC/Sources/Presentation/Modules/Splash/SplashView.swift [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        ZStack {
+            R.image.background.image
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+
+            R.image.logoWhite.image
+                .aspectRatio(contentMode: .fill)
+                .frame(maxWidth: .infinity)
+                .opacity(
+                    isLoaded ? 1 : 0
```
Comment (suho) [inline iosApp/PhongKMMIC/Sources/Presentation/Modules/Splash/SplashView.swift]:
```suggestion
                    isLoaded ? 1.0 : 0.0
```

**Review by suho — CHANGES_REQUESTED** [review]

**Comment by suho** [comment]
@phongvhd93 Please follow [this section](https://nimblehq.co/compass/development/code-reviews/author-guidelines/#handle-comments-of-code-reviewers) to handle comments (e.g., comment the commit to comment)

File: iosApp/PhongKMMIC/Sources/Presentation/Modules/Splash/SplashView.swift (hunk 3 above)
Comment (phongvhd93) [reply iosApp/PhongKMMIC/Sources/Presentation/Modules/Splash/SplashView.swift]: Solved in commit acb241c

**Comment by phongvhd93** [comment]
> @phongvhd93 Please follow [this section](https://nimblehq.co/compass/development/code-reviews/author-guidelines/#handle-comments-of-code-reviewers) to handle comments (e.g., comment the commit to comment)

Thank you, I got it

**Review by markgravity — APPROVED** [review]

**Review by suho — APPROVED** [review]

**Pull request merged**